### DISCUSSION TITLE
DNS updates for custom domain not working

### DISCUSSION THREAD

**Discussion by fmanaa** [swellstores/community#75] [Help] [answered]
I am trying to connect a custom domain to my website, it's been over 48 hours after updating DNS settings with GoDaddy for both the CNAME on the WWW record and the forwarding from my naked domain to WWW subdomain.

`CNAME WWW XYZ.apps.swellstores.com`, where XYZ is my domain name. Swell portal says to connect CNAME to XYZ.apps.swellstores.com, which is what I've done.

All I get is a 404 Not Found - NGINX when doing XYZ.com in a browser.

Note GoDaddy doesn't accept @ host for CNAME, hence why I had to do the domain forwarding instead.

What am I missing?

**Comment by jesseswell** [comment]
Hi @fmanaa !

Can you share a screenshot that includes the relevant records? 

From my experience, it could be something with SSL conflict causing the 404 not found error. To be sure, I need to see the record from your DNS Provider.

**Reply by fmanaa** [reply]
Here are my dns records:
![image](https://user-images.githubusercontent.com/2783563/211462540-d3879d8e-ed32-4c2d-b04f-5bc14e12705c.png)

I am also forwarding my naked domain to www subdomain using 301 redirect.

**Comment by jesseswell** [comment, marked as answer]
@fmanaa  did you add the www root when you added your custom domain in your Swell dashboard? Is auto-SSL active? 🤔

**Reply by fmanaa** [reply]
No I hadn't actually, only had the naked domain added on Swell. I just changed it to www and it worked like a charm! Amazing, thank you!
